instrumentation/instagocb/README.md

```diff
@@ -0,0 +1,131 @@
+# Instana instrumentation of Couchbase SDK v2 (gocb) for Go
+
+This module contains the code for instrumenting the Couchbase SDK, based on the [`gocb`](https://github.com/couchbase/gocb) library for Go.
+
+The following services are currently instrumented:
+
+1. Cluster
+   1. Query
+   2. SearchQuery
+   3. AnalyticsQuery
+2. Bucket
+3. Bucket Manager
+   1. Get/Create/Update/Drop/Flush Bucket
+4. Collection
+   1. Bulk Operations
+   2. Data operation (Insert, Upsert, Get etc)
+   3. Sub-doc operations
+      1. LookupIn & MutateIn
+5. Scope
+   1. Query
+   2. AnalyticsQuery
+6. Binary Collection
+7. Collection Manager
+8. Collection Data Structures
+   1. List
+   2. Map
+   3. Queue
+   4. Set
+
+> [!NOTE]
+> While you can call methods such as QueryIndexManager or SearchIndexManager from the provided instagocb cluster interface, it's important to note that tracing support for these methods is currently not implemented. If you find the instrumentation of any unlisted features necessary, please feel free to raise an issue.
+
+
+
+Compatibility
```

Comment:
Do we need this section?

- gocb version : It will affect if the auto update script updates the instrumentation pacakge.
- go version: This will also add a possible place for error if anyone who updates the package, forgets to update the README.

Reply:
Good point. I will remove this from the readme.